### ISSUE TITLE
make release on OSX

### ISSUE THREAD

**Issue by villins** [ysbaddaden/shards#40]
```
/usr/local/bin/crystal build --release -o bin/shards src/shards.cr --link-flags "-static -L/opt/crystal/embedded/lib"
ld: warning: directory not found for option '-L/opt/crystal/embedded/lib'
ld: library not found for -lcrt0.o
clang: error: linker command failed with exit code 1 (use -v to see invocation)
Error: execution of command failed with code: 1: `cc -o "bin/shards" "${@}" -static -L/opt/crystal/embedded/lib -rdynamic  -lyaml -levent -lpcl -lpcre -lgc -lpthread -ldl`
make: *** [release] Error 1
```

**Comment by howdoicomputer** [comment]
I bet this has to do with setting the LIBRARY_PATH variable.

**Comment by howdoicomputer** [comment]
I was able to do it without the link flags.

```
➜  shards git:(master) crystal build --release -o bin/shards src/shards.cr
```

**Comment by villins** [comment]
without the link flags, the result like ```make``` command，just build executable file

**Comment by howdoicomputer** [comment]
You're right. I'm tired. Nobody listen to me.

**Comment by ysbaddaden** [comment]
It's still a bit different. It's not a static build anymore, but it has `--release` which optimizes the built shard.

This make command is just a quick way for me to build the linux tarball on Linux for each release. I have no idea if static is required on OSX, but it should target another directory (I have no idea which one).

It would be nice to have the command building a nice tarball for OSX for #36, if anybody can provide this.

**Comment by ysbaddaden** [comment]
fixed in cba8260a45892de7926b127189f3b871f95310d6
